__tests__/workers/postUpdated.ts

```diff
@@ -459,7 +459,11 @@ it('should save a new post with content curation', async () => {
   });
   const posts = await con.getRepository(Post).find();
   expect(posts.length).toEqual(3);
-  expect(posts[2].contentCuration).toStrictEqual(['news', 'story', 'release']);
+  expect(posts[2].contentCuration).toIncludeSameMembers([
```

Comment:
Leaving it here, but did you mean to remove so many test and test files?

Reply:
Double checked them and they're relevant only for v1 so yes